Jenkinsfile

```
@@ -1,15 +1,17 @@
 //def buildConfiguration = buildPlugin.recommendedConfigurations()
 
-def lts = "2.176.1"
-def weekly = "2.199"
+def lts = "2.249.3"
+def weekly = "2.266"
 def buildConfiguration = [
+/*
   [ platform: "linux",   jdk: "8", jenkins: lts, javaLevel: "8" ],
   [ platform: "windows", jdk: "8", jenkins: lts, javaLevel: "8" ],
   [ platform: "linux",   jdk: "11", jenkins: lts, javaLevel: "8" ],
   [ platform: "windows", jdk: "11", jenkins: lts, javaLevel: "8" ],
+*/
   // Also build on recent weekly
-//  [ platform: "linux",   jdk: "11", jenkins: weekly, javaLevel: "8" ],
-//  [ platform: "windows", jdk: "11", jenkins: weekly, javaLevel: "8" ]
+  [ platform: "linux",   jdk: "11", jenkins: weekly, javaLevel: "8" ],
+  [ platform: "windows", jdk: "11", jenkins: weekly, javaLevel: "8" ]
```

`javaLevel` should be considered deprecated, and you should not specify `jenkins` unless you _overriding_ the POM version with something newer. Just delete the variables at the top.

```suggestion
  [ platform: "linux",   jdk: "11" ],
  [ platform: "windows", jdk: "11" ]
```